src/timessquare/storage/noteburstjobstore.py

```diff
@@ -0,0 +1,135 @@
+"""A Redis-based cache of outstanding noteburst notebook execution jobs."""
+
+from __future__ import annotations
+
+from typing import Any, Mapping, Optional
+
+import aioredis
+
+from timessquare.domain.noteburstjob import NoteburstJobModel
+
+from .redisutils import calculate_key
+
+
+class NoteburstJobStore:
```

Comment:
`NoteburstJobStore` and `NbHtmlCacheStore` look essentially identical to me, apart from a few parameters. Consider having them both be instances of a parameterized generic base class similar to:

https://github.com/lsst-sqre/gafaelfawr/blob/main/src/gafaelfawr/storage/base.py#L24

I think you could get rid of a fair bit of duplicate code that way.